airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -616,6 +616,17 @@ def depends_float(
     FilterParam[bool | None],
     Depends(filter_param_factory(DagModel.is_paused, bool | None, filter_name="paused")),
 ]
+QueryHasImportErrorsFilter = Annotated[
+    FilterParam[bool | None],
+    Depends(
+        filter_param_factory(
+            DagModel.has_import_errors,
+            bool | None,
+            filter_name="has_import_errors",
+            description="Filter DAGs by having import errors. Only DAGs that have been successfully loaded before will be returned.",
```

Comment:
```suggestion
            description="Filter Dags by having import errors. Only Dags that have been successfully loaded before will be returned.",
```

We are replacing `DAG` with `Dag` in the codebase.

Reply:
Good callout, I just saw the email thread. Updated.

Reply:
And based on what I read, it seems like this discussion is still on going? Looks like most people don't like DAGs, but there's no conclusion yet between `Dags` and `dags`.

Reply:
Yes indeed, you get it right. IMO, the frequency of "Dag" are still higher than "dag" in recent PR. Using "Dag" here _should_ be fine.